HelloBuilderCodeGenTests/Doubles/GroupBuilder.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 ACartagena. All rights reserved.
+
+import Foundation
+
+import Foundation
+@testable import HelloBuilderCodeGen
+
+extension Group: AutoTestBuilder {
+    static private let membersTestDefault: [User] = {
+        guard let user = try? UserBuilder().with(email: "me@email.com").build() else { return [] }
+        return [user]
+    }()
+    static private let nameTestDefault = "Group"
+}
+
+// sourcery:inline:Group.AutoTestBuilder
+final class GroupBuilder {
+	private var members: [User]? = []
```

Comment:
out of curiosity, if we set default values in builder pattern means our `build()` function doesn't need to be throwable?
```
private var members: [User]!
private var name: String!
```